packages/web/modals/add-liquidity.tsx

```diff
@@ -1,12 +1,14 @@
-import { FunctionComponent } from "react";
-import { observer } from "mobx-react-lite";
 import { ObservableAddLiquidityConfig } from "@osmosis-labs/stores";
-import { useStore } from "../stores";
-import { useConnectWalletModalRedirect, useAddLiquidityConfig } from "../hooks";
-import { AddLiquidity } from "../components/complex/add-liquidity";
-import { ModalBase, ModalBaseProps } from "./base";
+import { observer } from "mobx-react-lite";
+import { FunctionComponent } from "react";
 import { useTranslation } from "react-multi-lang";
+
+import { AddConcLiquidity } from "../components/complex/add-conc-liquidity";
```

Comment:
We could use `next/dynamic` to do a dynamic import of this component, to keep visx and other code out of the main bundle.

Example: https://github.com/osmosis-labs/osmosis-frontend/blob/825873b112d261332704b9c04bb4fad41f38c5c1/packages/web/modals/bridge-transfer.tsx#L14